apps/api_web/lib/api_web/views/route_pattern_view.ex

```diff
@@ -25,7 +25,14 @@ defmodule ApiWeb.RoutePatternView do
     :sort_order
   ])
 
+  defp fetch_representative_trip(trip_id) do
+    case State.Trip.by_id(trip_id) do
```

Comment:
- This is `List.first`: `trip_id |> State.Trip.by_id() |> List.first()`
- tests?

Reply:
Are representative trips guaranteed to be primary?

Reply:
At least one of them will be. Non-primary trips are those we duplicate based on their presence in `multi_route_trips.txt`